ingest_api/runtime/src/schemas.py

```diff
@@ -102,9 +90,9 @@ class AuthResponse(BaseModel):
 
 class WhoAmIResponse(BaseModel):
     sub: str = Field(..., description="A unique identifier for the user")
-    cognito_groups: List[str] = Field(
-        ..., description="A list of Cognito groups the user belongs to"
-    )
+    # cognito_groups: List[str] = Field(
+    #     ..., description="A list of Cognito groups the user belongs to"
+    # )
```

Comment:
We can pare this model down even more and allow the auth response from Cognito to add optional fields if needed using `model_config = ConfigDict(extra='allow')`